src/main/scala/io/kf/etl/processors/participantcommon/transform/step/MergePhenotype.scala

```diff
@@ -47,7 +47,9 @@ object MergePhenotype {
         }
 
         // Only append to source text in the positive case for observed
-        val sourceText = if (snomedObserved.nonEmpty || hpoObserved.nonEmpty) phenotype.source_text_phenotype else None
+        val isPhenotypeObserved = snomedObserved.nonEmpty || hpoObserved.nonEmpty || observedOpt.contains(true)
```

Comment:
This, `observedOpt.contains(true)`, is an extra condition that I ve added to make some tests pass. All in all, I think it makes sense to add it since it is the best test to determine if the phenotype is observed or not.

Reply:
Ok mais dans le fonds tu n'as pas besion de is `isPhenotypeObserved`. Si c'est observed ou non tu va l'avoir dans `observedOpt` non?